docs/install/kubernetes.md

```diff
@@ -35,7 +35,21 @@ curl -sSLo kubectl-storageos.tar.gz \
 > 💡 You can find binaries for different architectures and systems in [kubectl
 > plugin](https://github.com/storageos/kubectl-storageos/releases).
 
-### Install Ondat
+### Option A: Install Ondat (embedded etcd)
+
+```bash
+kubectl storageos install \
+    --include-etcd \
+    --etcd-tls-enabled \
+    --admin-username "myuser" \
+    --admin-password "my-password"
+```
+
+> 💡 This is the easiest way to get up and running with Ondat, though
```

Comment:
Given that the production recommendation is still "bring-your-own etcd", I would put that as an Option A?